### ISSUE TITLE
bug hotwaire ?

### ISSUE THREAD

**Issue by evanillaa** [qbcore-framework/qb-vehiclekeys#37]
if IsPedInAnyVehicle(ped, false) and lockpicked and not IsHotwiring and not HasKey then

If all the cars on the road are closed, the doors are closed
You can't open the door unless you kill the driver
In this case, you cannot use hotwaire

Cars parked on the road
You can open it with a lockpick
You can start the vehicle without doing anything
The vehicle runs without anything
But it is correct after unlocking the vehicle with the lockpick
To attempt to operate the vehicle through qb-skillbar or hotwire

This is where the work gets a bit good
and suitable for everyone
Because everyone knows these ways

While you are in the trial period, do not restart the script while you are inside the vehicle
